s3/replication/common/src/s3replicationcommon/s3_put_object_tagging.py

```diff
@@ -127,6 +127,9 @@ async def send(self):
                     self._logger.info('Response headers {}'.format(
                         self._response_headers))
 
+                    # Delete temporary tagset file.
+                    os.system('rm -rf tagset.xml')
```

Comment:
I may have missed that we are using a temporary file to prepare the tagset. We should find another way.

Reply:
okay.
Here, I am removing a temporary tagset file which is a duplicate of the original tagset file. 
Original object_tagset.xml file is still there in the config folder.